src/controller/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -411,6 +411,88 @@ CHIP_ERROR ExampleOperationalCredentialsIssuer::GenerateNOCChain(const ByteSpan
     return CHIP_NO_ERROR;
 }
```

Comment:
All these files (ExampleCredentialIssuerCommands.h, ExampleOperationalCredentialsIssuer.cpp, ExampleOperationalCredentialsIssuer.h) are used by chip-tool. jf-control-app uses the files from [examples/jf-control-app/commands/example](https://github.com/project-chip/connectedhomeip/tree/master/examples/jf-control-app/commands/example). Please move the changes in the correct place.

Reply:
This has been addressed in the latest commit.

Reply:
I think all the changes from src/controller can be removed - I am not sure that there is any benefit in exposing the JF PKI interfaces there. Feel free to remove those changes or to resolve this conversation if you disagree.